code/controllers/subsystems/job.dm

```diff
@@ -167,9 +167,12 @@ SUBSYSTEM_DEF(jobs)
 					continue
 
 				var/age = V.client.prefs.age
-
-				var/min_job_age = job.get_minimum_character_age(V.get_species())
-				var/ideal_job_age = job.get_ideal_character_age(V.get_species())
+				//handles cases such as if head is IPC, where age is not really a factor
+				var/min_job_age = 1
+				var/ideal_job_age = 20
+				if(!(V.get_species() in ALL_IPC_SPECIES))
+					min_job_age = job.get_minimum_character_age(V.get_species())
+					ideal_job_age = job.get_ideal_character_age(V.get_species())
```

Comment:
This doesn't seem clean. Why not add IPC ages to the jobs themselves? This way, different jobs can have different ages. This is already done for skrell for instance.

Reply:
if I do that, it would triple the length of the 'ideal species' since there are like 8 different IPC chassises, if devs want it that way, I can handle it to be as such

Reply:
also they would all be the same regardless